content/en/docs/languages/java/configuration.md

```diff
@@ -138,6 +226,9 @@ customize:
 The logging exporter prints the name of the span along with its attributes to
 stdout. It's mainly used for testing and debugging.
 
+> This configuration option applies when using the
+> `opentelemetry-exporter-logging` artifact.
```

Comment:
A reader really needs the group id and the version, in addition to the artifact id, to make work. I wonder if should link all of these to the [listing of artifacts](https://github.com/open-telemetry/opentelemetry-java?tab=readme-ov-file#releases) we maintain in the root of the project.